src/exodus/createpayload.h

```diff
@@ -39,5 +39,6 @@ std::vector<unsigned char> CreatePayload_MetaDExCancelEcosystem(uint8_t ecosyste
 std::vector<unsigned char> CreatePayload_ExodusAlert(uint16_t alertType, uint32_t expiryValue, const std::string& alertMessage);
 std::vector<unsigned char> CreatePayload_DeactivateFeature(uint16_t featureId);
 std::vector<unsigned char> CreatePayload_ActivateFeature(uint16_t featureId, uint32_t activationBlock, uint32_t minClientVersion);
+std::vector<unsigned char> CreatePayload_CreateDenomination(uint32_t propertyId, uint64_t value);
```

Comment:
I believe type of `value` should be int64_t instead because it's need to be compatible with `CAmount`.

Reply:
Same as previous issue. If we want to fix this one we should fix the other too, which need a lot of effort so I think we should fix it later.

Reply:
agree